include/hlslib/common/OpenCL.h

```diff
@@ -634,17 +856,156 @@ class Buffer {
 
  private:
 #ifdef HLSLIB_XILINX
-  ExtendedMemoryPointer CreateExtendedPointer(void *hostPtr,
-                                              MemoryBank memoryBank) {
-    ExtendedMemoryPointer extendedPointer;
-    extendedPointer.flags = BankToFlag(memoryBank, true);
-    extendedPointer.obj = hostPtr;
-    extendedPointer.param = 0;
-    return extendedPointer;
-  }
+   ExtendedMemoryPointer CreateExtendedPointer(void *hostPtr,
```

Comment:
I just noticed, something is fucked up with the indentation here, this and a lot below is indented with 3 spaces (??). Could you check what happened?

Reply:
Hmm clang-format seems to be confused if not the whole class to format is selected it seems